ocs/message.go

```diff
@@ -63,6 +73,17 @@ func (m *TalkRoomMessageData) PlainMessage() string {
 	return tr
 }
 
+// DisplayName returns the display name for the sender of the message (" (Guest)" is appended if sent by a guest user)
+func (m *TalkRoomMessageData) DisplayName() string {
+	if m.ActorType == ActorGuest {
+		if m.ActorDisplayName == "" {
+			return "(Guest)"
```

Comment:
I would not use the brackets here, its what the UI also does and otherwise this results in `<(Guest)>` in the matterbridge

Reply:
Makes sense